frontend/cypress/e2e/talentsearch/search-workflows.cy.js

```diff
@@ -8,89 +11,308 @@ describe("Talent Search Workflow Tests", () => {
+      searchRejectsMySingleCandidate();
+      // reset
+      cy.wrap(combobox).type("{backspace}");
+      searchFindsMySingleCandidate();
+      // pass
+      cy.wrap(combobox).type("Ontario{enter}");
       cy.wait("@gqlCountApplicantsAndCountPoolCandidatesByPoolQuery");
+      searchFindsMySingleCandidate();
+    });
 
+    // working language ability - fail
+    cy.findByRole("radio", {
+      name: /French only/i,
+    }).click();
+    cy.wait("@gqlCountApplicantsAndCountPoolCandidatesByPoolQuery");
+    searchRejectsMySingleCandidate();
 
-      cy.wrap($input).type("Ontario{enter}{enter}")
-      cy.wait("@gqlCountApplicantsAndCountPoolCandidatesByPoolQuery");
+    // working language ability - pass
+    cy.findByRole("radio", {
+      name: /English only/i,
+    }).click();
+    cy.wait("@gqlCountApplicantsAndCountPoolCandidatesByPoolQuery");
+    searchFindsMySingleCandidate();
 
+    // employment duration - fail
+    cy.findByRole("radio", {
+      name: /Term duration/i,
+    }).click();
+    cy.wait("@gqlCountApplicantsAndCountPoolCandidatesByPoolQuery");
+    searchRejectsMySingleCandidate();
 
-      cy.wrap($input).type("National Capital{enter}{enter}")
-      cy.wait("@gqlCountApplicantsAndCountPoolCandidatesByPoolQuery");
+    // employment duration - pass
+    cy.findByRole("radio", {
+      name: /Indeterminate duration/i,
+    }).click();
+    cy.wait("@gqlCountApplicantsAndCountPoolCandidatesByPoolQuery");
+    searchFindsMySingleCandidate();
 
+    // employment equity, no negation possible
+    cy.findByRole("checkbox", {
+      name: /Woman/i,
+    }).click();
+    cy.wait("@gqlCountApplicantsAndCountPoolCandidatesByPoolQuery");
+    searchFindsMySingleCandidate();
 
-      cy.wrap($input).type("Atlantic{enter}{enter}");
-      cy.wait("@gqlCountApplicantsAndCountPoolCandidatesByPoolQuery");
+    // skills selection, not currently used in search
+    cy.get("@testSkill").then((skill) => {
+      cy.findByRole("button", {
+        name: `Add this skill : ${skill.name.en}`,
+      }).click();
+      // skill selection does not trigger an api request
+      searchFindsMySingleCandidate();
+    });
 
-    })
+    // conditions of employment, no negation possible
+    cy.findByRole("checkbox", {
+      name: /ability to work overtime \(Occasionally\)/i,
+    }).click();
+    cy.wait("@gqlCountApplicantsAndCountPoolCandidatesByPoolQuery");
+    searchFindsMySingleCandidate();
 
-    searchReturnsGreaterThanZeroApplicants();
+    // check total user count
+    // no way to know what the exact number should be without resetting the database, so just check it's not zero
+    cy.findByRole("heading", {
+      name: /Results: [1-9][0-9]* matching candidate/i,
+    });
 
-    cy.findAllByRole("button", { name: /Request Candidates/i }).then($button => {
-      cy.wrap($button)
-        .should("exist")
-        .and("be.visible")
-        .and("not.be.disabled");
+    cy.findAllByRole("article", {
```

Comment:
```suggestion
    cy.findByRole("article", {
```

Reply:
Good catch!  I've changed this to findByRole in https://github.com/GCTC-NTGC/gc-digital-talent/pull/4998/commits/44f59935d54487c1cab50f88ab2dd90f738fdcc8.